apps/storybook-e2e/src/e2e/components/checkbox/vl-checkbox.stories.cy.ts

```diff
@@ -5,6 +5,7 @@ describe('story vl-checkbox default', () => {
     it('should be accessible', () => {
         cy.visitWithA11y(checkboxUrl);
 
+        cy.get('vl-checkbox');
```

Comment:
waarom overal `cy.get('[component]')` voor elke `cy.checkA11y()`?
zal die `checkA11y` sowieso niet falen als het component niet aanwezig is?

of is dit iets dat moest om te testen te laten werken in nieuwe opzet